src/firefly_exchange_client.py

```diff
@@ -746,9 +747,9 @@ async def get_user_account_data(self):
         """
             Returns user account data.
         """
-        return self.apis.get(
+        return await self.apis.get(
             service_url = SERVICE_URLS["USER"]["ACCOUNT"],
-            query = '',
+            query = {},
```

Comment:
nitpick
if query is passed as an empty dictionary over here. ideally it would be better if on api_service.py it is set by default to an empty dictionary then we dont need to pass anything here. but its just a nitpick doesnt effect code in any way.